File: MediATech/Model/DAL/User/IUserDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MediATech.Model.DAL.User
{
    public interface IUserDAO
    {
        public bool Login(string email, string password);

        public bool Register(string firstName, string lastName, string email, string password);
    }
}

File: src/main/java/mediatech/Model/DAL/User/IUserDAO.java
package mediatech.Model.DAL.User;

import mediatech.Model.BL.User;

public interface IUserDAO {
    
    public boolean register(User user);
    
    public boolean login(String email, String password);
    
    public boolean updatePassword(String email, String newPassword);
    
    public User getUser(String email, String password);
    
    public boolean close();
}
